data_fixes/AC107/create_new_top_containers.rb

```diff
@@ -0,0 +1,55 @@
+require 'archivesspace/client'
+require 'active_support/all'
+require 'json'
+require_relative '../../helper_methods.rb'
+
+aspace_staging_login()
+
+start_time = "Process started: #{Time.now}"
+puts start_time
+
+csv = CSV.parse(File.read("AC107_Create_New_Containers.csv"), :headers => true)
+log = "log_create_top_containers.txt"
+
+top_containers = []
+#get values from the input csv
+csv.each do |row|
+  #ils_holding_id = row['ils_holding_id']
+  @repo = row['repo']
```

Comment:
```suggestion
  repo = row['repo']
```
to be consistent with the other columns?

Reply:
Thanks @sandbergja ! I need the repo again in the subsequent loop, hence the `@`.